### DISCUSSION TITLE
sys_crash

### DISCUSSION THREAD

**Discussion by GHDavor** [sultim-t/xash-rt#66] [General]
hello, i'm having a problem with the rt, the problem is that the game doesn't launch, it throws an sys_crash error with some code and some dlls, any help?
![the error](https://user-images.githubusercontent.com/91034453/221370999-c734e40b-c465-4978-adee-43c3c575bd9c.png)

**Comment by zorkiii** [comment]
same here

**Reply by GHDavor** [reply]
@sultim-t pls help

**Comment by FlaGnuM** [comment]
Same problem here, I have i5-8400 + GTX 1060 6GB, is it not enough even to play at 30FPS ?
![image](https://user-images.githubusercontent.com/126399333/221415045-5f440d45-a4eb-4bba-99fa-4c813f128ecb.png)

**Reply by GHDavor** [reply]
i also have an 1060, but its 12gb so it should be runnable, i played quake 2 rtx and i had around 30 or 40 fps so, idk

**Reply by FlaGnuM** [reply]
Hey I got it to work thanks to the guy below u just have to download https://github.com/sultim-t/xash-rt/files/10931917/RayTracedGL1.zip and replace it in rt_bin.
and i was right lol, i was able to run the game with the "Ultra Performance" AMD FSR settings at 1080p with constant +30FPS and sometimes (rarely) it will even hit 40 at certain closed spaces 
but what bugs me still is why the hell do i have a 1% cpu usage at 99% of the time lol 
![image](https://user-images.githubusercontent.com/126399333/225782944-1a93f807-e460-4b7f-92ca-5e232932d929.png)
![image](https://user-images.githubusercontent.com/126399333/225782951-4be0d5fa-2e8b-4e03-9f51-5d0f4953d0e0.png)

**Reply by yunyxn** [reply]
i was able to finish the game with the patch at mostly 40 fps using fsr performance mode on a 1080 and it was pretty enjoyable all around. hoping fsr 3 can breathe projects like these more life onto our ancient cards

**Reply by FlaGnuM** [reply]
@yunyxn yeah if only DLSS wasn't exclusive to the RTX series we maybe could've had better performance or at least more options to mess with !

**Comment by GHDavor** [comment]
@sultim-t we really want to play this, help pls

**Comment by PixelThePixel** [comment]
Same problem. Any idea how to fix it? I also a Gtx 1060

**Comment by Cracky64** [comment]
Got the Same Error with my Intel ARC 770 LE 16GB - I think mainly "real" RTX GeForce Cards are Supported for some reason.

**Reply by GHDavor** [reply]
but rtx cards are getting the same error

**Comment by HotBolt** [comment]
GTX 1070 here, the same problem. Is there any way to make it work?

**Comment by emesvau** [comment]
Same problem here on an Arc A770 w/ newest drivers on Windows 11.

<img width="316" alt="hlrt_error" src="https://user-images.githubusercontent.com/127204233/223410292-00764dbc-64b4-45ca-9a80-b09cab08a2e3.png">

**Reply by yunyxn** [reply]
could you email me? i've got a possible fix pushed as a PR at https://github.com/sultim-t/RayTracedGL1/pull/31 that i couldn't verify with an arc gpu. thanks in advance

**Comment by emesvau** [comment]
Hey there.Sure, how can I help you… ***@***.*** is my mail.RegardsMalte Von meinem iPhone gesendetAm 08.03.2023 um 23:32 schrieb yuni ***@***.***>:﻿
could you email me? i've got a possible fix pushed as a PR at sultim-t/RayTracedGL1#31 that i couldn't verify with an arc gpu. thanks in advance

**Comment by yunyxn** [comment]
github seems to have trimmed the email out. please let me know if the game
launches with this in the rt_bin folder (replace the original)
https://drive.google.com/file/d/13XMIBXH-fNM4tP4GSisaNOqclMFuHy3-/view?usp=drive_web

**Reply by emesvau** [reply]
Hello there.

Just tested it with your .dll, unfortunately it didn't help.

![grafik](https://user-images.githubusercontent.com/127204233/223976579-b72bef50-ecc5-47e6-a32c-fd16d9481bdc.png)

**Reply by yunyxn** [reply]
thanks for letting me know, ill try something else in a few hours

**Reply by yunyxn** [reply]
please let me know if this still crashes
[RayTracedGL1.zip](https://github.com/sultim-t/xash-rt/files/10931917/RayTracedGL1.zip)

**Reply by Cracky64** [reply]
Still the Same with this .dll file. If i can add any information to help out, let me know! ![grafik](https://user-images.githubusercontent.com/118550713/224109783-45f37962-e072-437d-8834-ad24e1b517b0.png)

**Reply by yunyxn** [reply]
hmm i suppose arc has a different flaw entirely

**Reply by McGarfeld** [reply]
I don't know what to do, the files above didn't solve the problem
![BgyvOoo7W6Q](https://user-images.githubusercontent.com/128860121/227652041-2e31e74f-845f-4454-9276-ec4540878f30.jpg)
